tests/tracking/fluent/test_fluent.py

```diff
@@ -487,6 +487,12 @@ def test_start_run_defaults_databricks_notebook(
         assert is_from_run(active_run, MlflowClient.create_run.return_value)
 
 
+@pytest.mark.parametrize("experiment_id", [("a", "b"), {"a", "b"}, ["a", "b"], {"a": 1}])
```

Comment:
Let's test some falsy objects (e.g. empty list).

Reply:
good idea! Validated class existence in the check `if experiment_id is not None` :)